pkg/portutil/port_allocate_linux.go

```diff
@@ -96,6 +99,7 @@ func portAllocate(protocol string, ip string, count uint64) (uint64, uint64, err
 			}
 		}
 		if needReturn {
+			allocateStart = int(start + count)
```

Comment:
is this related to container inspect warning ?

Reply:
No, I merged main and squashed commits.

Reply:
> is this related to container inspect warning ?

Sorry I messed up a bit. I'll close this one and push a new.